spacy/lang/char_classes.py

```
@@ -4,23 +4,23 @@
 import regex as re
 
 re.DEFAULT_VERSION = re.VERSION1
-merge_char_classes = lambda classes: '[{}]'.format('||'.join(classes))
-split_chars = lambda char: list(char.strip().split(' '))
-merge_chars = lambda char: char.strip().replace(' ', '|')
+merge_char_classes = lambda classes: "[{}]".format("||".join(classes))
```

Ruff (0.12.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `merge_char_classes` as a `def`

(E731)